server/apps/immich/src/api-v1/asset/asset-repository.ts

```diff
@@ -150,6 +154,27 @@ export class AssetRepository implements IAssetRepository {
     return builder.getRawMany();
   }
 
+  async getAssetTimelineLayout(userId: string, dto: GetTimelineLayoutDto): Promise<AssetTimelineLayout[]> {
+    const builder = this.assetRepository
+      .createQueryBuilder('asset')
+      .leftJoin('asset.exifInfo', 'ei')
+      .select(`date_trunc('month', "fileCreatedAt")`, 'timeBucket')
+      .addSelect('ei.exifImageHeight', 'exifImageHeight')
+      .addSelect('ei.exifImageWidth', 'exifImageWidth')
+      .addSelect('ei.orientation', 'orientation')
+      .where('"ownerId" = :userId', { userId })
+      .andWhere('asset.isVisible = true')
+      .andWhere('asset.isArchived = false')
+      .andWhere('asset.resizePath is not NULL')
```

Comment:
I think you need to remove this one, since you have it conditionally in the if statement below now.